### PR TITLE
Balder/gc enforce rank profile inheritance

### PR DESCRIPTION
**Pull request by baldersheim**
@hmusum PR

### CODE REVIEW

File: container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java [hunk 1]
```diff
@@ -235,6 +239,11 @@ private void watchPortChange() {
         }
     }
 
+    void recofigure(QrConfig qrConfig) {
```
Comment (hmusum) [inline container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java:242]: Typo in method name

**Review by hmusum — APPROVED** [review]

File: container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java (hunk 1 above)
Comment (baldersheim) [reply container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java:242]: Thanks, fixed in #19293

Comment (baldersheim) [reply container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java:242]: Have to redo this one. The first part should not be there.

**Comment by baldersheim** [comment]
This is superseded by #19296

**Pull request closed without merging**